roleplay_economy.py

```diff
@@ -0,0 +1,1098 @@
+from red_star.plugin_manager import BasePlugin
+from red_star.command_dispatcher import Command
+from red_star.rs_errors import CommandSyntaxError, UserPermissionError
+from red_star.rs_utils import respond, RSArgumentParser, decode_json, split_output
+from discord import Embed, File, Message, Reaction, Forbidden, NotFound
+from io import BytesIO
+from difflib import SequenceMatcher as SM
+import shlex
+import json
+import re
+from copy import deepcopy
+from time import time
+
+
+class RoleplayEconomy(BasePlugin):
+    name = "roleplay_economy"
+    version = "1.0"
+    author = "GTG3000"
+
+    default_config = {
+        "shop_delay": 120
+    }
+
+    item_base = {
+        "name": "Default Item",
+        "description": "This item looks very generic",
+        "category": None,
+        "image": None,
+        "inshop": False,
+        "p_buy": 100,
+        "p_sell": 100,
```

Comment:
I feel like these should probably be a bit more verbose, something like "buy_price" and "sell_price"